### PR TITLE
docs(cni): Add more info to CNI docs

### PR DESCRIPTION
**Pull request by slonka**
Signed-off-by: slonka <slonka@users.noreply.github.com>

Expand CNI page in docs, add:
- different configurations
- logs location

Closed https://github.com/kumahq/kuma-website/issues/957

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: docs/docs/dev/networking/cni.md [hunk 1]
```diff
@@ -1,44 +1,270 @@
...
+  --set "cni.confName=10-calico.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/home/kubernetes/bin,cni.confName=10-calico.conflist \
+   kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:184]: To be consistent with other instructions where we are using multiple `--set` flages, I would go with:
```suggestion
helm install \
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.chained=true" \
  --set "cni.netDir=/etc/cni/net.d" \
  --set "cni.binDir=/home/kubernetes/bin" \
  --set "cni.confName=10-calico.conflist" \
   kuma kuma/kuma
```

File: docs/docs/dev/networking/cni.md [hunk 2]
```diff
@@ -1,44 +1,270 @@
...
+  --set "cni.containerSecurityContext.privileged=true"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.securityContext.privileged=true \
+   kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:217]:
```suggestion
helm install 
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.containerSecurityContext.privileged=true" \
    kuma kuma/kuma
```

Comment (bartsmykla) [reply docs/docs/dev/networking/cni.md:217]: In this suggestion I also changed `cni.securityContext` to `cni.containerSecurityContext` which is correct one

File: docs/docs/dev/networking/cni.md [hunk 3]
```diff
@@ -1,44 +1,270 @@
...
+  --set "cni.enabled=true"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true \
+   kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:240]:
```suggestion
helm install \
  --namespace kuma-system \
  --set "cni.enabled=true" \
    kuma kuma/kuma
```

File: docs/docs/dev/networking/cni.md [hunk 4]
```diff
@@ -1,44 +1,270 @@
...
+  --set "cni.confName=10-aws.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-aws.conflist \
    kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:155]:
```suggestion
helm install \
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.chained=true" \
  --set "cni.netDir=/etc/cni/net.d" \
  --set "cni.binDir=/opt/cni/bin" \
  --set "cni.confName=10-aws.conflist" \
    kuma kuma/kuma
```

File: docs/docs/dev/networking/cni.md [hunk 5]
```diff
@@ -1,44 +1,270 @@
...
+  --set "cni.confName=10-azure.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-azure.conflist \
+   kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:129]:
```suggestion
helm install \
  --create-namespace \
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.chained=true" \
  --set "cni.netDir=/etc/cni/net.d" \
  --set "cni.binDir=/opt/cni/bin" \
  --set "cni.confName=10-azure.conflist" \
    kuma kuma/kuma
```

File: docs/docs/dev/networking/cni.md [hunk 6]
```diff
@@ -1,44 +1,270 @@
...
+  --set "cni.netDir=/etc/cni/net.d" \
+  --set "cni.binDir=/opt/cni/bin" \
+  --set "cni.confName=10-kindnet.conflist"
+```
+
+:::
+::: tab "Helm"
 
 ```shell
 helm install --namespace kuma-system \
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:101]:
```suggestion
helm install \
  --create-namespace \
  --namespace kuma-system \
```

File: docs/docs/dev/networking/cni.md [hunk 7]
```diff
@@ -1,44 +1,270 @@
...
-  --set cni.enabled=true,cni.chained=true,cni.netDir="/etc/cni/net.d",cni.binDir=/opt/cni/bin,cni.confName=10-calico.conflist \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/etc/cni/net.d,cni.binDir=/opt/cni/bin,cni.confName=10-kindnet.conflist \
+   kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:103]:
```suggestion
  --set "cni.enabled=true" \
  --set "cni.chained=true" \
  --set "cni.netDir=/etc/cni/net.d" \
  --set "cni.binDir=/opt/cni/bin" \
  --set "cni.confName=10-kindnet.conflist" \
    kuma kuma/kuma
```

File: docs/docs/dev/networking/cni.md [hunk 8]
```diff
@@ -1,44 +1,270 @@
...
 ```
 
 :::
 ::: tab "Helm"
 
-When using [Helm](../../installation/helm), we should use the values in the `cni` section to set the relevant parameters.
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir=/var/lib/rancher/k3s/agent/etc/cni/net.d,cni.binDir=/bin,cni.confName=10-flannel.conflist \
+   kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:77]:
```suggestion
helm install \
  --create-namespace \
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.chained=true" \
  --set "cni.netDir=/var/lib/rancher/k3s/agent/etc/cni/net.d" \
  --set "cni.binDir=/bin" \
  --set "cni.confName=10-flannel.conflist" \
    kuma kuma/kuma
```

File: docs/docs/dev/networking/cni.md [hunk 9]
```diff
@@ -1,44 +1,270 @@
...
+  --set "cni.confName=10-calico.conflist"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --namespace kuma-system \
+  --set cni.enabled=true,cni.chained=true,cni.netDir="/etc/cni/net.d",cni.binDir=/opt/cni/bin,cni.confName=10-calico.conflist \
+   kuma kuma/kuma
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:51]:
```suggestion
helm install \
  --create-namespace \
  --namespace kuma-system \
  --set "cni.enabled=true" \
  --set "cni.chained=true" \
  --set "cni.netDir=/etc/cni/net.d" \
  --set "cni.binDir=/opt/cni/bin" \
  --set "cni.confName=10-calico.conflist" \
    kuma kuma/kuma
```

File: docs/docs/dev/networking/cni.md [hunk 10]
```diff
@@ -1,44 +1,270 @@
...
+
+# Kuma CNI v2
+
+The new version of the CNI is using new [kuma-net](https://github.com/kumahq/kuma-net/) engine to do transparent proxying.
+
+The new engine is capable of using the following methods to transparently redirect traffic:
+- iptables (default)
+- eBPF
+
+To use the new CNI with eBPF add `experimental.cni=true` and `experimental.ebpf.enabled=true` settings to either `kumactl` or `helm`.
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:257]:
```suggestion
To use the new CNI with eBPF add `experimental.ebpf.enabled=true` settings to either `kumactl` or `helm`.
```

**Comment by slonka** [comment]
This is the tree that is built:
image

File: docs/docs/dev/networking/cni.md [hunk 11]
```diff
@@ -1,44 +1,320 @@
...
+kumactl install control-plane \
+  --set "cni.enabled=true"
+```
+
+:::
+::: tab "Helm"
+
+```shell
+helm install --create-namespace --namespace kuma-system \
+  --set cni.enabled=true \
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:264]:
```suggestion
  --set "cni.enabled=true" \
```

File: docs/docs/dev/networking/cni.md [hunk 12]
```diff
@@ -1,44 +1,320 @@
...
+## Kuma CNI v2
+
+The v2 version of the CNI is using [kuma-net](https://github.com/kumahq/kuma-net/) engine to do transparent proxying.
+
+To install v2 CNI append the following options to the command from [installation](#installation):
+
+```
+--set ... \
+--set "cni.enabled=true" \
+--set "experimental.ebpf.enabled=true"
```
Comment (bartsmykla) [inline docs/docs/dev/networking/cni.md:287]:
```suggestion
--set "experimental.cni=true"
```

**Review by bartsmykla — APPROVED** [review]
lgtm

**Pull request merged**